client/shipping-label/views/purchase/steps/rates/list.js

```diff
@@ -63,7 +61,7 @@ const ShippingRates = ( {
 
 	return (
 		<div>
-			{ renderRateNotice( showRateNotice ) }
+			{ shouldShowRateNotice && renderRateNotice() }
```

Comment:
React will ignore non-0 falsey values, so if `shouldShowRateNotice` evaluates to `false` or `null`, it won't get rendered by React ([ref](https://facebook.github.io/react/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored)). There isn't a need to return an empty string in the false case, then.

The `&&` is a common pattern for condition rendering ([ref](https://facebook.github.io/react/docs/conditional-rendering.html#inline-if-with-logical--operator)), and we also use it frequently in Calypso, so would prefer to keep that.